fenix/app/src/main/java/org/mozilla/fenix/compose/BottomSheetHandle.kt

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.onClick
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * A composable for displaying a handle present on top of a bottom sheet. This is selectable when
+ * talkback is enabled.
+ *
+ * @param onRequestDismiss Invoked on clicking the handle when talkback is enabled.
+ * @param contentDescription Content Description of the composable.
+ * @param modifier The modifier to be applied to the Composable.
+ * @param color Color of the handle.
+ */
+@Composable
+fun BottomSheetHandle(
+    onRequestDismiss: () -> Unit,
+    contentDescription: String,
+    modifier: Modifier = Modifier,
+    color: Color = FirefoxTheme.colors.textSecondary,
```

Comment:
just to document this, the color will likely change. I assume this color was taken from the Tabs Tray Banner UI, but the color is legacy from the XML, and we'll probably switch to a more-appropriate color token eventually.

Reply:
gotcha, thanks for letting me know! We can update it here when that happens and maybe even remove the param if the design system component has the fixed color based on the theme. Also a opportunity to reuse the same component across the app.